prow/config/config.go

```diff
@@ -2234,6 +2234,10 @@ func parseProwConfig(c *Config) error {
 		return fmt.Errorf("tide has invalid max_goroutines (%d), it needs to be a positive number", c.Tide.MaxGoroutines)
 	}
 
+	if c.Tide.MaxGraphQLGoroutines < 0 {
```

Comment:
You could have the type be an unsigned integer, and the YAML marshaller will error this for you